packages/fxa-settings/src/components/images/graphic_hearts_broken.tsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+
+export const HeartsBroken = () => {
```

Comment:
Why do we need to do it this way rather than `import { ReactComponent as HeartsBroken } from './graphic_hearts_broken.svg';`, doesn't it end up being inlined anyway?

I guess it's not the worst thing having them here so it feels a little better modifying class names from within the TSX instead of the SVG, but I think I personally am okay with having class names on the SVG directly, it beats having classes and inline styles within them. 😅 These are all missing the license at the top, btw.

Reply:
I can try changing it back! At the time, I thought that it wasn't working as SVGs, but that might have actually been secondary.

Reply:
Yep can confirm -- adding the animations to the tailwind config and then adding the classes to the svg does not apply the animations. I think that it might be because tailwind treeshakes the output styles, and it doesn't scan the SVGs for the classnames, so it trims them out of the output? Maybe? But yeah it seems like we need to have these in ts

Reply:
But I will add the license in!

Reply:
> I think that it might be because tailwind treeshakes the output styles, and it doesn't scan the SVGs for the classnames, so it trims them out of the output? Maybe? But yeah it seems like we need to have these in ts

Ah, yes, you'll have to add SVGs to `config.content` to make that work, but that should be quick. 🙈 We do this in content-server's tailwind config file since there's some JS files that have classes in them.